simulations/vip-193/vip-193/simulations.ts

```diff
@@ -0,0 +1,75 @@
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers";
+import { expect } from "chai";
+import { ethers } from "hardhat";
+
+import { expectEvents, initMainnetUser } from "../../../src/utils";
+import { forking, testVip } from "../../../src/vip-framework";
+import { CORE_MARKETS, vip193 } from "../../../vips/vip-193/vip-193";
+import MOCK_TOKEN_ABI from "./abi/MOCK_TOKEN_ABI.json";
+import VTOKEN_ABI from "./abi/VTOKEN_ABI.json";
+
+const PROTOCOL_SHARE_RESERVE = "0xCa01D5A9A248a830E9D93231e791B1afFed7c446";
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+let vToken: ethers.Contract;
+let underlying: ethers.Contract;
+let impersonatedTimelock: SignerWithAddress;
+const provider = ethers.provider;
+
```

Comment:
Why aren't we storing and checking in this simulation `preVipStorage` and `postVipStorage`?